backend/x86_gas.ml

```diff
@@ -288,10 +288,17 @@ let print_line b = function
       bprintf b "\t.file\t%d\t\"%s\""
         file_num (X86_proc.string_of_string_literal file_name)
   | Indirect_symbol s -> bprintf b "\t.indirect_symbol %s" s
-  | Loc (file_num, line, col) ->
+  | Loc { file_num; line; col; discriminator } ->
       (* PR#7726: Location.none uses column -1, breaks LLVM assembler *)
+      (* CR gyorsh for mshinwell: If we don't set colmun, it will have the value
+         set for column in the previous .loc direcitve.
+         Should this fix be upstreamed? *)
```

Comment:
That sounds like a bug.  Could you remove this CR and make a separate (flambda-backend) PR?  We can think about how to upstream some of these in due course.

Reply:
I removed this change from the current PR and submitted it as a separate PR #7.